content/en/docs/reference/using-api/server-side-apply.md

```diff
@@ -465,12 +465,18 @@ kubectl apply --server-side --field-manager=my-manager [--dry-run=server]
 
 ## API Endpoint
 
-With the Server Side Apply feature enabled, the `PATCH` endpoint accepts the
-additional `application/apply-patch+yaml` content type. Users of Server Side
+With the Server-Side Apply feature enabled, the `PATCH` endpoint accepts the
+additional `application/apply-patch+yaml` content type. Users of Server-Side
 Apply can send partially specified objects as YAML to this endpoint.  When
 applying a configuration, one should always include all the fields that they
 have an opinion about.
 
+### RBAC And permissions
```

Comment:
```suggestion
### RBAC and Permissions
```
or
```suggestion
### RBAC and permissions
```

Reply:
@apelisse , What do you think about the second suggestion?

Reply:
Thanks for the ping, missed that somehow, sending a fix with the second suggestion!